docs/sources/datasources/azuremonitor/_index.md

```diff
@@ -0,0 +1,255 @@
+| Authentication          | Enables Managed Identity. Selecting Managed Identity will hide many of the fields below. See [Configuring using Managed Identity](#configuring-using-managed-identity) for more details.                                                                                                                                |
+| Azure Cloud             | The national cloud for your Azure account. For most users, this is the default "Azure". For more information, see [the Azure documentation.](https://docs.microsoft.com/en-us/azure/active-directory/develop/authentication-national-cloud)                                                                             |
+| Directory (tenant) ID   | The directory/tenant ID for the Azure AD app registration to use for authentication. See [Get tenant and app ID values for signing in](https://docs.microsoft.com/en-us/azure/active-directory/develop/howto-create-service-principal-portal#get-tenant-and-app-id-values-for-signing-in) from the Azure documentation. |
+| Application (client) ID | The application/client ID for the Azure AD app registration to use for authentication.                                                                                                                                                                                                                                  |
+| Client secret           | The application client secret for the Azure AD app registration to use for authentication. See [Create a new application secret](https://docs.microsoft.com/en-us/azure/active-directory/develop/howto-create-service-principal-portal#option-2-create-a-new-application-secret) from the Azure documentation.          |
+| Default subscription    | _(optional)_ Sets a default subscription for template variables to use                                                                                                                                                                                                                                                  |
+| Default workspace       | _(optional)_ Sets a default workspace for Log Analytics-based template variable queries to use                                                                                                                                                                                                                          |
+
+## Azure Monitor query editor
+
+The Azure Monitor data source has three different modes depending on which Azure service you wish to query:
+
+- **Metrics** for [Azure Monitor Metrics](https://docs.microsoft.com/en-us/azure/azure-monitor/essentials/data-platform-metrics)
+- **Logs** for [Azure Monitor Logs](https://docs.microsoft.com/en-us/azure/azure-monitor/logs/data-platform-logs)
+- [**Azure Resource Graph**](https://docs.microsoft.com/en-gb/azure/governance/resource-graph/overview)
```

Comment:
Azure Resource Graph should be bold.

Reply:
Also, I suggest adding hyperlinks to topics like "Querying Azure Monitor Metrics", "Querying Azure Monitor Logs" and "Querying with Azure Resource Graph" upfront. Otherwise, users may not know the Querying Azure Monitor Logs section exists.

Reply:
Azure Resource Graph is bold 😅

Good suggestion for the links though